### PR TITLE
Create data generation file

### PR DESCRIPTION
**Pull request by ryancmoniz**
Data generation script creates 10,000,000 primary records(games), 10000 users, and a random amount of reviews(1-40) per primary record.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,13 +23,16 @@
   "homepage": "https://github.com/Team-Gandalf/reviews-service#readme",
   "dependencies": {
     "axios": "^0.19.2",
+    "basho-riak-client": "^2.4.0",
```
Comment (Gen0pp) [inline package.json:26]: Could remove this if you're done with it

File: neo4j/csv_writer.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      steam_level: faker.random.number(1000),
+    });
+    console.log(`User # ${i} of ${USERS} successfully written.`);
+  }
+  userWriter.end();
+  console.log('done');
+};
+
+generateGames();
+generateUsers();
```
Comment (Gen0pp) [inline neo4j/csv_writer.js:83]: Could add a 'console.time('took');' and 'console.timeEnd('took');' if you want some metrics.

Comment (ryancmoniz) [reply neo4j/csv_writer.js:83]: Thanks, I was thinking about a way to time my generation!

File: package.json (hunk 1 above)
Comment (ryancmoniz) [reply package.json:26]: Definitely!

**Pull request merged**